tests/payloads/call_log_payload.json

```diff
@@ -0,0 +1,148 @@
+            "district": null,
+            "district_audio": null,
+            "dost_cc": null,
+            "dost_flow_status": "P1M10D5done",
+            "dost_module10_streak": null,
+            "dost_module5_streak": null,
+            "dost_module7_streak": null,
+            "dost_nudge_group": null,
+            "dost_program": "Program1",
+            "dost_program_completion": null,
+            "dost_signup_status": "Yes",
+            "dost_time": "0",
+            "dost_ttr_group": null,
+            "dutystation": null,
+            "edulanguage": null,
+            "email": null,
+            "engagementcounter": null,
+            "feedback": null,
+            "fl_a10_results": null,
+            "fl_a11_results": null,
+            "fl_a12_results": null,
+            "fl_a1_results": null,
+            "fl_a2_results": null,
+            "fl_a3_results": null,
+            "fl_a4_results": null,
+            "fl_a5_results": null,
+            "fl_a6_results": null,
+            "fl_a7_results": null,
+            "fl_a8_results": null,
+            "fl_a9_results": null,
+            "gender": null,
+            "gram_panchayat": null,
+            "last_incoming_call": null,
+            "lastactivedate": null,
+            "level": null,
+            "lname": null,
+            "location_coordinates": null,
+            "mobiletype": null,
+            "mothername": null,
+            "pin": null,
+            "pincode": null,
+            "polls_completed": null,
+            "polls_responded": null,
+            "pregnantmothername": null,
+            "program": null,
+            "projectname": null,
+            "role": null,
+            "sector": null,
+            "sm_contractual_status": null,
+            "sm_current_condition": null,
+            "sm_date_of_reporting": null,
+            "sm_dependent_current_condition": null,
+            "sm_email": null,
+            "sm_gender": null,
+            "sm_mobile_number": null,
+            "sm_name": null,
+            "sm_no_of_dependent_affected": null,
+            "sm_office": null,
+            "sm_patient_state": null,
+            "sm_section": null,
+            "sm_title": null,
+            "staff_type": null,
+            "stafftype": null,
+            "state": null,
+            "stream": null,
+            "team": null,
+            "test_response_result": null,
+            "time": null,
+            "title": null,
+            "totalnochild": null,
+            "type_of_participant": null,
+            "updnumber": null,
+            "ups": null,
+            "upweek1": null,
+            "ward": null,
+            "week": null,
+            "whohavephone": null,
+            "whose_affected": null
+        },
+        "groups": [
+            {
+                "name": "Dost-Pilot-Intro",
+                "uuid": "4bf04450-cf03-4eed-8e25-f7e6f0dd3301"
+            }
+        ],
+        "name": "",
+        "urn": "tel:+919761033697",
```

Comment:
I think adding a payload with someone's phone number will not be good. You can use a random phone number instead of it.